spec2vec/Spec2Vec.py

```diff
@@ -20,31 +23,59 @@ class Spec2Vec(BaseSimilarity):
     Example code to calcualte spec2vec similarities between query and reference
     spectrums:
 
-    .. code-block:: python
+    .. testcode::
 
+        import os
         import gensim
         from matchms import calculate_scores
+        from matchms.filtering import add_losses
+        from matchms.filtering import default_filters
+        from matchms.filtering import normalize_intensities
+        from matchms.filtering import require_minimum_number_of_peaks
+        from matchms.filtering import select_by_intensity
+        from matchms.filtering import select_by_mz
+        from matchms.importing import load_from_mgf
         from spec2vec import Spec2Vec
-        from spec2vec import SpectrumDocument
 
-        # reference_spectrums & query_spectrums loaded from files using https://matchms.readthedocs.io/en/latest/api/matchms.importing.load_from_mgf.html
-        references = [SpectrumDocument(s, n_decimals=2) for s in reference_spectrums]
-        queries = [SpectrumDocument(s, n_decimals=2) for s in query_spectrums]
+        def spectrum_processing(s):
+            '''This is how a user would typically design his own pre- and post-
+            processing pipeline.'''
+            s = default_filters(s)
+            s = normalize_intensities(s)
+            s = select_by_mz(s, mz_from=0, mz_to=1000)
+            s = select_by_intensity(s, intensity_from=0.01)
+            s = add_losses(s, loss_mz_from=10.0, loss_mz_to=200.0)
+            s = require_minimum_number_of_peaks(s, n_required=5)
+            return s
 
-        # Import pre-trained word2vec model (alternative: train new model)
-        model_file = "path and filename"
+        spectrums_file = os.path.join(os.getcwd(), "..", "tests", "pesticides.mgf")
```

Comment:
It will be hard to find this file reading this on https://spec2vec.readthedocs.io/en/latest/. Can you add a download link to the example intro text with 
```rst
:download:`../tests/pesticides.mgf </../tests/pesticides.mgf>`
:download:`../integration-tests/test_user_workflow_spec2vec.model </../integration-tests/test_user_workflow_spec2vec.model>`
```
So people dont have to go GitHub to find the file.

Reply:
I will try adding this.

Reply:
Added this to the docstring. But then also realized I have no admin rights on readthedocs so I couldn't inspect the results.

Reply:
You can build the docs locally with `cd readthedocs;make html;google-chrome _build/html/index.html`.

Added you as maintainer of https://readthedocs.org/dashboard/spec2vec/users/

Reply:
Perfect, thanks Stefan.
Don't know why I didn't think of rendering it locally.... but that of course works nicely :)
(and thanks for adding me as admin).